infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLog.java

```diff
@@ -19,6 +20,12 @@
      */
     void append(long address, LogData entry);
 
+    /**
+     * Append a list consecutive entries ordered by their addresses.
+     * @param entries
+     */
```

Comment:
add more comments on the behavior or writing to trimmed or already written addresses.

Reply:
done